charts/consul/test/terraform/gke/main.tf

```diff
@@ -45,6 +45,10 @@ resource "google_container_cluster" "cluster" {
   }
   subnetwork      = data.google_compute_subnetwork.subnet.name
   resource_labels = var.labels
+
+  release_channel {
+    channel = "RAPID"
```

Comment:
```suggestion
    channel = "rapid"
```

Reply:
Does lowercase work? https://cloud.google.com/kubernetes-engine/docs/concepts/release-channels#updating_the_cluster_release_channel

Reply:
or is it `release_channel = "rapid"` instead of the whole stanza?

Reply:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#release_channel